src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.model;
+
+import racingcar.util.CarNameValidator;
+
+public class Car {
+
+    private final String carName;
+    private int moveCount = 0;
+
+    public Car(String carName, Integer moveCount) {
+
+        CarNameValidator.validate(carName);
+        this.carName = carName;
+        this.moveCount = moveCount;
+    }
+
+    public Car(String carName) {
+
+        CarNameValidator.validate(carName);
+        this.carName = carName;
```

Comment:
이것은 고려안해도 되지만 validate의 검증을 해서 리턴값으로 다시 돌려주는 방법도 좋을 거 같아요! 하지만, 분할하여 가독성을 높이는 방법도 괜찮다 생각합니다!

```suggestion
        this.carName =  CarNameValidator.validate(carName);
```